.gitignore

```diff
@@ -1,8 +1,9 @@
 # Documentation is generated into this directory
 /docs
 
-# Do not commit package build files to github
-/packages/core/dist
+# Do not commit package build files other than the compiled CSS
+/packages/core/dist/**/
```

Comment:
I think an ideal solution would have the Release .zip have the dist files while also preventing dist files from being in GH. Related to [this ticket](https://jira.cms.gov/browse/WNMGDS-267)

Reply:
This issue seems relatively high priority to fix, but I think it still might be worth looking into release.zip before reverting this .gitignore